docs/api/static-transfers.md

```diff
@@ -0,0 +1,79 @@
+# Static Transfer API
+
+The Static transfer API is a non-interactive API for downloading LFS objects.
+Its main intent is to provide simple interoperability with existing data
+storage solutions by offloading serving LFS objects to them while using
+git and git LFS for version management.
+
+Support for it by Git LFS clients and servers is optional.
+
+Git LFS clients MAY implement this transfer API in a fallback manner:
+first attempting to download the LFS object using the Static transfer
+API and if this fails (e.g., the manifest file does not exist,
+oid cannot be found in the manifest file) use the Basic transfer API.
+
+## Downloads
+
+Downloading an object first requires fetching a manifest file from using
+HTTP from the remote server. The proposed location of the manifest file
+is `https://<remote domain>/.well-known/git-lfs-manifest.json`. The
+location is configured using the `lfs.staticurl` configuration option,
+or remote-specific `remote.{name}.lfsstaticurl` configuration option.
+Both of them can be provided in `.lfsconfig` file as well.
+
+The manifest file MUST contain a JSON payload that looks like this:
+
+```json
+{
+  "version": "1",
+  "transfer": "static",
+  "objects": [
+    {
+      "oid": "1111111",
+      "size": 123,
+      "authenticated": true,
```

Comment:
Let's note that `authenticated` needs to have its normal value.  That is, this may not be set, and a user may need to authenticate using the normal mechanisms.

Reply:
If we do not want expiration of tokens here, then we should probably just remove `authenticated` as well, no?

Reply:
I don't think we need to, because if the repository is public and doesn't require authentication, then `authenticated` can be `true`.

Reply:
Ah, I see. I misunderstood this flag. So `authenticated=true` means that entry is already authenticated and nothing extra has to be done to obtain the file. File `authenticated=false` means they have to authenticate with other means. Sounds good.